src/test/shared/debug/launchConfiguration.test.ts

```diff
@@ -99,14 +99,15 @@ describe('LaunchConfiguration', function () {
     const workspace = vscode.workspace.workspaceFolders![0]
     const testLaunchJson = vscode.Uri.file(path.join(workspace.uri.fsPath, '.vscode/launch.json'))
     /** Object representation of the launch.json test file. */
-    const testLaunchJsonData = JSON.parse(testutil.fromFile(testLaunchJson.fsPath))
+    let testLaunchJsonData: any
```

Comment:
```suggestion
    let testLaunchJsonData: string
```

Reply:
JSON.parse returns an `any` type implicitly, anything else causes the code that uses this to have errors.